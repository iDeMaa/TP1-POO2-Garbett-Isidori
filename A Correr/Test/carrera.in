16	4	2	13
13	17
18	29
30	49
50	80
10	20
21	70
15	F
19	M
34	M
20	F
64	F
53	F
23	M
18 	M
12	M
42	F
29	F
17	M
22	F
61	F
26	F
57	M
5
2
7
4
8
11
3
13
1
9
15
12
16