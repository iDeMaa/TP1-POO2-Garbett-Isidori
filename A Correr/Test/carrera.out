1	1	0	0
2	4	11	13
3	0	0	0
4	5	0	0
1	2	8	9
2	7	3	16